static/src/js/components/Tour/SearchTour.jsx

```diff
@@ -3,12 +3,18 @@ import React, {
   useEffect,
   useContext
 } from 'react'
+import { useSelector } from 'react-redux'
 import Joyride, { STATUS, ACTIONS } from 'react-joyride'
-import TourSteps, { TOTAL_STEPS } from './TourSteps'
+import TourSteps from './TourSteps'
 import TourContext from '../../contexts/TourContext'
+import { isLoggedIn } from '../../util/isLoggedIn'
 
 const SearchTour = () => {
   const { runTour, setRunTour } = useContext(TourContext)
+  const authToken = useSelector((state) => state.authToken)
```

Comment:
A more common way that we do this is that we'd have a container component like in `static/src/js/containers/SecondaryToolbarContainer/SecondaryToolbarContainer.jsx` render that in app.jsx and then pass down the authToken which we'd pull out of the redux state there

Reply:
That is true, but I think with our introduction of useContext with this component I like matching that paradigm and using a hook to accomplish the same thing here.